app/assets/stylesheets/modals.scss

```diff
@@ -1,4 +1,4 @@
-.move-workspace, .duplicate-workspace {
+.workspace-form {
```

Comment:
This seemed better than adding `.new-workspace` since we currently have no need to discriminate.